### ISSUE TITLE
Is the issue in Chrome "undefined" - the editor - caused by PyV*

### ISSUE THREAD

**Issue by phangs** [emmetio/livestyle-sublime-old#105]
Hi, Every time I try to use LiveStyle it gets broken, I mean the panel shows "undefined" where the editor name should be.  I have noticed that even removing LiveStyle in Package control doesn't fix the problem, what I do to have LS working again with Chrome is to manually delete the PyV8 folder in the Installed Package directory under the \AppData\Roaming\Sublime Text 3 folder then re-install Emmet and LS in the Package Controller.  Doing this every time LS breaks in ST3 fixes my problem.

So, is this issue in LS caused by PyV8.  Anyway, just giving some info about this issue.

**Comment by sergeche** [comment]
As I said earlier, you are using very outdated LiveStyle plugins: PyV8 is not used by LS for a few months already

**Comment by phangs** [comment]
I'm sorry but I don't get it. How come I am using a very outdated Livestyle plugin if I am installing it directly in ST3 using Package Control as stated in the install instructions?  Sorry if you are getting annoyed.

**Comment by MattHill72** [comment]
In defense of Phang, your install instructions do explain that you need to install PyV8 manually if it's not installed automatically.

http://livestyle.emmet.io/install/

Perhaps that bit needs to be removed from the instuctions if it's causing confusion.

**Comment by sergeche** [comment]
@phangs maybe you have a proxy or something?

**Comment by phangs** [comment]
@sergeche @MattHill72 thanks, yeah I did follow the install intructions by installing PyV8 manually the first time, PyV8 doesn't install automatically every time I install Livestyle.  What I am doing now is to remove Emmet plugin and PyV8 in the Package Control, restart ST3 and re-install Emmet so I could get a fresh PyV8 so I don't have to do it manually.  I have to do this every time I close ST3 to get Livestyle working with Chrome.

@sergeche I'm not behind any proxies but we do use OpenDNS in our network, could that be the cause?  Anyway, thank you for your patience, much appreciated.  I'll just probably continue with my procedure until I found out what is causing this issue on my setup.  

Best regards.
